### ISSUE TITLE
JAWS+Firefox ignores content in double-nested iframe elements

### ISSUE THREAD

**Issue by patrickhlauke** [FreedomScientific/standards-support#728]
### Summary

When an HTML document includes an `<iframe>`, which in turn includes another nested `<iframe>`, the content of the actual content inside that second-level `<iframe>` is completely ignored by JAWS. Content itself can't be reached with reading keys, and other functionality - such as bringing up the headings list - also does not include anything in that double-nested `<iframe>`.

This problem only appears in JAWS+Firefox (tested with latest versions: Windows 11 using Firefox 115.0 (64-bit) and JAWS 2023.2306.38). JAWS+Chrome, NVDA+Firefox all work correctly.

### Expected result

content inside the second-level `<iframe>` should be exposed/reachable using JAWS.

### Actual result

content inside the second-level `<iframe>` is completely hidden from JAWS.

### Example

Test case available as a zip file at https://mega.nz/file/5GYlxJjQ#Ny19YYKUTBbPyzbwYFZFdAWKK15HlTwYkJcFDqCX9cI

The zip contains two tests:

* `jaws-firefox-single-nested-iframe`: open the index in Firefox; the content of the single iframe can be reached/accessed as normal
* `jaws-firefox-double-nested-iframe`: open the index...and all of a sudden JAWS fails to see/reach the content in the double-nested iframe

Included also a screen recording of both cases - here, for clarity, I even open JAWS' heading list window. In the single iframe case, it lists the content heading as well. In the double-nested case, the innermost heading is not listed at all.

https://github.com/FreedomScientific/standards-support/assets/895831/eaa6baf6-148c-446e-9954-61c5a71e09d8

#### JAWS version and build number

2023.2306.38
 
#### Operating System and version

Windows 11

#### Browser and version:

Firefox 115.0

**Comment by LJWatson** [comment]
Thanks for filing this @stevefaulkner. Note that this problem was also evident in Firefox 114, however I do not believe it was evident in the version of Jaws that precedes the one you reference. @patrickhlauke can you confirm since you were involved in the initial discussion about this issue?

**Comment by patrickhlauke** [comment]
@LJWatson ah yes, apologies. from memory, this was indeed not a problem in previous versions of JAWS (when I first tested the specific page that led us to this issue, I was running an earlier version, and then updated to the latest version of JAWS. I stupidly didn't note down the previous version number though)

**Comment by stevefaulkner** [comment]
can reproduce

**Comment by tonystecca** [comment]
Hi. This issue is affecting a project I'm working on. What's the status of this defect?

**Comment by stevefaulkner** [comment]
@tonystecca it has been assigned to @BrettLewisVispero he will be able to provide and update

**Comment by BrettLewisVispero** [comment]
I couldn't reproduce this with JAWS 2024 and Firefox Version 125.0.3
I think Mozilla must have fixed something on their side.

**Comment by patrickhlauke** [comment]
Confirming that using Firefox 126.0 and JAWS 2024.2405.83
